beetsplug/export.py

```diff
@@ -40,6 +42,9 @@ class ExportPlugin(BeetsPlugin):
 
     def __init__(self):
         super(ExportPlugin, self).__init__()
+        # Used when testing export plugin
+        self.run_results = None
+        self.export_format = None
```

Comment:
I also don't quite see why this is a class field instead of just being a local variable in the `run` method. Can we please revert to the old way?

Reply:
I reverted it.